modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/admin/catalog/resource/v1_0/test/BaseOptionValueResourceTestCase.java

```diff
@@ -329,6 +341,16 @@ public void testDeleteOptionValue() throws Exception {
 			optionValueResource.deleteOptionValueHttpResponse(
 				optionValue.getId()));
 
+		assertHttpResponseStatusCode(
```

Comment:
Hey @jalber786 why are we generating twice the same check? shouldn't once be enough?